dev/docs/source/exceptions.rst

```diff
@@ -129,7 +129,7 @@ Raises::
         must be <= 31 chars.
 
 Or for a worksheet name containing one of the Excel restricted characters,
-i.e. ``' [ ] : * ? / \'``::
+i.e. ``[ ] : * ? / \``::
```

Comment:
I agree that this is a bit unclear in the rendered text. Particularly since it starts, but doesn't end, with an extra space.